lib/helpers/document/getSubdocumentStrictValue.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function getSubdocumentStrictValue(parts, schema) {
```

Comment:
it would probably help to add jsdoc for this, so it can be understood more easily at a glance